inventory.yml.sample

```diff
@@ -101,6 +101,9 @@ all:
     # The retrieved cluster kubeconfig will be placed on the bastion host at the following location
     kubeconfig_dest_dir: /home/redhat/
     kubeconfig_dest_filename: "{{ cluster_name }}-kubeconfig"
+    kubeadmin_dest_filename: "{{ cluster_name }}-kubeadmin.vault.yml"
```

Comment:
`kubeadmin_dest_filename` is exposed in the inventory, but the var is not used anywhere. 

It should be referenced in `post_install.yml` in this var: https://github.com/redhat-partner-solutions/crucible/blob/c982fdbeaf7b051c16e08c84313876b205cf435b/post_install.yml#L11

Reply:
:facepalm:

Reply:
Fixed now